### DISCUSSION TITLE
GITHUB AUTH ERROR

### DISCUSSION THREAD

**Discussion by fenibofubara** [adonisjs/core#3074] [Help]
Hi guys, how do i fix this reoccuring error when trying to authenticate via github. Am currently running on adonis 4.1.0

{
"status": false,
"message": "Unable to authenticate. Try again later",
"hint": "E_OAUTH_STATE_MISMATCH: Oauth state mis-match"
}

**Comment by marsyeen** [comment]
It could be your running the wrong oauth urls or don't have SSL valid certificate. I've ran into this issue once.
